cmake/kconfig.cmake

```diff
@@ -18,26 +18,14 @@ set(ENV{KERNELVERSION}      ${PROJECT_VERSION})
 set(ENV{KCONFIG_CONFIG}     ${DOTCONFIG})
 set(ENV{KCONFIG_AUTOHEADER} ${AUTOCONF_H})
 
-set(kconfig_target_list
-  config
-  gconfig
-  menuconfig
-  oldconfig
-  xconfig
-  )
-
-set(COMMAND_FOR_config     ${KCONFIG_CONF} --oldaskconfig ${KCONFIG_ROOT})
-set(COMMAND_FOR_gconfig    gconf                          ${KCONFIG_ROOT})
+set(kconfig_target_list menuconfig)
```

Comment:
I wonder if we need a target list anymore? Could we simplify this? Maybe a follow-up PR?